build.boot

```diff
@@ -3,18 +3,21 @@
   :resource-paths #{"resources"}
   :dependencies '[[org.clojure/clojure "1.9.0" :scope "provided"]
                   [adzerk/boot-cljs "2.1.4" :scope "test"]
+                  [adzerk/boot-cljs-repl   "0.3.3"]
+                  [com.cemerick/piggieback "0.2.1"  :scope "test"]
+                  [weasel                  "0.7.0"  :scope "test"]
+                  [org.clojure/tools.nrepl "0.2.12" :scope "test"]
                   [adzerk/boot-reload "0.5.2" :scope "test"]
                   [pandeiro/boot-http "0.8.3" :scope "test"]
                   [javax.xml.bind/jaxb-api "2.3.0" :scope "test"] ; necessary for Java 9 compatibility
                   ; project deps
                   [org.clojure/clojurescript "1.10.238" :scope "test"]
-                  [nightlight "RELEASE"]
```

Comment:
We might need to add this back as a dependency, I'm getting an error about nightlight stuff not being found so I think play-cljs is still referencing it somewhere